docs/using-mina/how-to-use-zkapp.mdx

```diff
@@ -2,4 +2,39 @@
 title: How to use a zkApp
 ---
 
-Describe
+import Subhead from '@site/src/components/common/Subhead';
+
+# Install a Wallet
+
+<Subhead>Learn how to interact with a zero-knowledge smart contract</Subhead>
+
+:::info
```

Comment:
Should appear above the H1, for consistency with other pages.

Reply:
@iregina  I see now... the zkApp docs have more solid examples of placement for the header

see https://github.com/o1-labs/docs2/blob/main/docs/zkapps/how-to-test-a-zkapp.mdx for a good example, move to line 6